frontend/src/pages/create-account.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    
+    <!--#include virtual="includes/head.html" -->
+    
+    <body>
+        <!--#include virtual="includes/header.html" -->
+        <div class="large-12 columns pnl-title pnl-summary">
+            <h1>Register</h1>
+            <div class="large-8 row">
+                <form id="sign-up" method="POST" action="">
+                    <div class="small-12 large-6 column">
+                        <label for="fname">First name:</label>
+                        <input type="text" name="fname" placeholder="First Name" required>
+                    </div>
+                    <div class="small-12 large-6 column">
+                        <label for="sname">Surname name:</label>
+                        <input type="text" name="sname" placeholder="Surname" required>
+                    </div>
+                    <div class="small-12 large-6 column">
+                        <label for="email">Email:</label>
+                        <input type="email" name="email" placeholder="Email" required>
```

Comment:
For consistency, whatever the placeholder for the login.html `Email` field should be the same here. [See PR #5](url) 
![image](https://cloud.githubusercontent.com/assets/11095605/13056101/157f6d50-d40b-11e5-817f-a689cdf3c746.png)